File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.cs
public class Solution {
    public int[] solution(int[] arr) {
        if (arr.Length == 1) {
            return new int[] { -1 };
        }

        int mini = arr[0];
        int miniIndex = 0;

        for (int i = 1; i < arr.Length; i++) {
            if (mini > arr[i]) {
                mini = arr[i];
                miniIndex = i;
            }
        }

        int[] answer = new int[arr.Length - 1];
        int index = 0;
        for (int i = 0; i < arr.Length; i++) {
            if (i != miniIndex) {  
                answer[index++] = arr[i];
            }
        }

        return answer;
    }
}

File: 프로그래머스/1/12935. 제일 작은 수 제거하기/제일 작은 수 제거하기.py
def solution(arr):
    answer = []
    min_value = min(arr)
    """
    아니 파이썬에는 min() 함수가 있다고요?
    min = arr[0]
    for i in range(len(arr)):
        if(min > arr[i]):
            min = arr[i]
    """
    for i in range(len(arr)):
        if(arr[i] != min_value):
            answer.append(arr[i])
    if len(answer) != 0:
        return answer
    else:
        return [-1]

        